ui/src/app/pages/access/data-access-requirements.tsx

```diff
@@ -360,27 +361,17 @@ const handleRasCallback = (
   spinnerProps: WithSpinnerOverlayProps,
   reloadProfile: Function
 ) => {
-  const handler = withErrorModal({
-    title: 'Error saving identity verification status.',
-    message:
-      'An error occurred trying to save your identity verification status. Please try again.',
-    onDismiss: () => {
```

Comment:
It does not appear like the spinner was working before. Leaving the behavior as it was.